rsync_to_rdisc.py

```diff
@@ -78,7 +78,7 @@ def send_mail_incomplete(run, title_template, subject, run_file):
 def check_rsync(run, folder, temperror, log):
     if not Path(temperror).stat().st_size:
         msg_bgarray_log = [[""], [">>> No errors detected <<<"]]
-        Path(temperror).unlink()  # remove tmperror file.
+        Path(temperror).unlink()
```

Comment:
if the file does not exist for any reason, an exception will be thrown and the app will crash since it's not being caught. Is this covered elsewhere in the code?

Reply:
No it is not covered. However, temperror will be created for each rsync process. Although not impossible, it is unlikely the file will not exist.